python/common/org/python/types/FrozenSet.java

```diff
@@ -124,4 +124,11 @@ public FrozenSet(org.python.Object[] args, java.util.Map<java.lang.String, org.p
     public org.python.Object __not_contains__(org.python.Object other) {
         return new org.python.types.Bool(!this.value.contains(other));
     }
+
+    @org.python.Method(
+            __doc__ = "Return a shallow copy of a FrozenSet."
+    )
+    public org.python.Object copy() {
+        return this;
```

Comment:
This isn't a *copy* - it's an internal reference. If you have a set, then copy it, modifying the original shouldn't modify the copy, or vice versa.